app.js

```diff
@@ -253,8 +253,15 @@ app.view("conduct_report", async ({ ack, view, client }) => {
   }
 });
 
-app.command("/prevreports", async ({ command, ack, client }) => {
+app.command("/prevreports", async ({ command, ack, client, respond }) => {
   await ack();
+  if(!ALLOWED_CHANNELS.includes(command.channel_id)) {
+  respond({
+text: `You are not in the correct channel for this :P`,
```

Comment:
Pretty sure you need to `await` respond here, also please fix the formatting :heavysob:

Reply:
whoops i did this on github file editor

Reply:
oop well its merged to late :P